src/constants.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+exports.PROTOCOL_MULTICODEC = '/rendezvous/1.0.0'
+exports.MAX_NS_LENGTH = 255
```

Comment:
This isn't mentioned anywhere in the spec, it should be, at least as a recommendation otherwise we risk compat issues.